### DISCUSSION TITLE
How to change position image cover in V2 config

### DISCUSSION THREAD

**Discussion by CRAG666** [tramhao/termusic#403] [Q&A]
my config v1:
[album_photo_xywh]
x_between_1_100 = 0
y_between_1_100 = 97
width_between_1_100 = 32
align = "BottomLeft"
my config v2: 
align = "bottom left"
x_between_1_100 = 0
y_between_1_100 = 97
size_scale = 90
hidden = false

**Comment by CRAG666** [comment]
Or save position
![2024-12-29T16:25:41,601414997-06:00](https://github.com/user-attachments/assets/488d1191-ac93-48a6-ac9a-5ee27a7ea220)
Error with sixel

**Comment by hasezoey** [comment]
>  my config v2:

The V2 coverart settings only have 3 fields:
https://github.com/tramhao/termusic/blob/12f28f075cf0afb32db60baa6e83e9d6da3a27c4/lib/src/config/v2/tui/mod.rs#L93-L100

This had been changed as i did not fully understand what each field did and thought that instead of exact positions (which could change with terminal size) to just have 4 alignments, hidden state and image scale.
Though this part (in the TUI side in the display) could need some refactoring as they still operate on the old values and the new ones are just translated there.

For now, you should be able to set `align = "bottom right"` and at least get the image full on-screen without artifacts.
